preprocessing.py

```diff
@@ -77,8 +74,6 @@ def write_tf_examples(filename, tf_examples, serialize=True):
             else:
                 writer.write(ex)
 
-# Read tf.Example from files
```

Comment:
why are you removing these?

Reply:
They are no longer adjacent to the functions they describe.